### PR TITLE
Adaptations to 2 RGW pods

### PR DESCRIPTION
**Pull request by ebenahar**
Starting OCS 4.5, we have 2 RGW pods running instead of 1 

Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/resources/storage_cluster.py [hunk 1]
```diff
@@ -205,11 +205,11 @@ def ocs_install_verification(
     )
 
     # rgw check only for VmWare
-    if config.ENV_DATA.get('platform') == constants.VSPHERE_PLATFORM:
+    if config.ENV_DATA.get('platform') in constants.ON_PREM_PLATFORMS:
         assert pod.wait_for_resource(
             condition=constants.STATUS_RUNNING,
             selector=constants.RGW_APP_LABEL,
-            resource_count=1,
+            resource_count=2,
```
Comment (prsurve) [inline ocs_ci/ocs/resources/storage_cluster.py:212]: We should have to check for old ocs version as well

Comment (ebenahar) [reply ocs_ci/ocs/resources/storage_cluster.py:212]: yes, will do that

**Comment by prsurve** [comment]
looks like there are few more places we need to add rgw pod check 

1. https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/mcg.py#L136-L144
2. https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ocs_upgrade.py#L118-L121

**Comment by ebenahar** [comment]
> looks like there are few more places we need to add rgw pod check
> 
> 1. https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/mcg.py#L136-L144
> 2. https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ocs_upgrade.py#L118-L121

Great catch!

**Pull request merged**